app/models/devise_token_auth/concerns/user.rb

```diff
@@ -1,15 +1,13 @@
 # frozen_string_literal: true
 
-require 'bcrypt'
```

Comment:
user shouldn't care about encryption, so 💪

Reply:
What do you mean?

Reply:
this is a positive comment, don't need to anything about. I meant that you removed the encryption/tokenization from the user model which follows SRP